src/pages/Room/SearchSongs/SearchSongs.js

```diff
@@ -123,14 +124,20 @@ const SearchSongs = ({ room: { id: roomId, style: roomStyle } }) => {
         onClick={() => clearSearch()}
       />
     ) : (
-      <FontAwesomeIcon
-        icon="heart"
-        title={intl.formatMessage({
+      <TooltipGate
+        content={intl.formatMessage({
           id: 'app.pages.Room.SearchSongs.FavouriteSongsTooltip',
         })}
-        className="get-favourite-songs"
-        onClick={() => getFavouriteSongs()}
-      />
+        place="bottom"
+      >
+        <span>
+          <FontAwesomeIcon
```

Comment:
This element has a `margin-right`, so whenever you hover the mouse over it, the tooltip looks misplaced. Please, take care of it.

![Captura de pantalla 2021-04-10 192812](https://user-images.githubusercontent.com/8061481/114279041-e838a200-9a32-11eb-9681-2c1883aec663.png)